Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -496,19 +462,22 @@ class TabTrayController: UIViewController {
 
     @objc
     private func didTapMenu() {
-        let mvc = MenuViewController(withAppState: .TabTray(tabTrayState: self.tabTrayState), presentationStyle: .Popover)
+        let presentationStyle: MenuViewPresentationStyle = (self.traitCollection.horizontalSizeClass == .Compact && traitCollection.verticalSizeClass == .Regular) ? .Modal : .Popover
```

Comment:
seeing as the toolbar is always along the bottom of the screen, never at the top, do we ever want to display the menu as popover, even in iPad/Landscape?